src/docs/reference/modules/Liquid/README.md

```diff
@@ -504,8 +504,19 @@ The following properties are available on the `Culture` object.
 
 | Property | Example | Description |
 | --------- | ---- |------------ |
-| `Name` | `en-US` | The request's culture as an ISO language code. |
+| `Name` | `en-US` | The ISO language code of the current culture. |
 | `Dir` | `rtl` | The text writing direction. |
+| `DisplayName` | `English (United States)` | The display name of the current culture. |
+| `NativeName` | `English (United States)` | The native name of the current culture. |
+| `TwoLetterISOLanguageName` | `en` | The two-letter ISO language name of the current culture. |
+
+##### supported_cultures filter
+
+Returns the currently supported cultures.
+
+```liquid
+{{ Culture | supported_cultures }}
+```
```

Comment:
Note that the suggestion apparently can't include too many ` characters, so the trailing fence is missing but is there in the code if you try to edit this comment:

```suggestion
Returns the currently supported cultures. Here is an example of how to print the names of supported cultures using a list:

```liquid
{% assign cultures = Culture | supported_cultures %}
<ul>
{% for culture in cultures %}
    <li>{{ culture.Name }}</li>
{% endfor %}
</ul>
```
```